src/elements/content-preview/PreviewNavigation.js

```diff
@@ -27,28 +38,45 @@ const PreviewNavigation = ({ collection = [], currentIndex, intl, onNavigateLeft
     }
 
     return (
-        <>
-            {hasLeftNavigation && (
-                <PlainButton
-                    className="bcpr-navigate-left"
-                    onClick={onNavigateLeft}
-                    title={intl.formatMessage(messages.previousFile)}
-                    type="button"
-                >
-                    <IconNavigateLeft />
-                </PlainButton>
-            )}
-            {hasRightNavigation && (
-                <PlainButton
-                    className="bcpr-navigate-right"
-                    onClick={onNavigateRight}
-                    title={intl.formatMessage(messages.nextFile)}
-                    type="button"
-                >
-                    <IconNavigateRight />
-                </PlainButton>
-            )}
-        </>
+        <NavRouter history={history}>
+            <Route path={['/:activeTab/:deeplink', '/']}>
+                {({ match }) => (
+                    <>
+                        {hasLeftNavigation && (
+                            <PlainButton
+                                className="bcpr-navigate-left"
+                                onClick={() => {
+                                    if (match.params.deeplink) {
+                                        history.push(`/${match.params.activeTab}`);
+                                    }
```

Comment:
Due to the added complexity of deeplink, you can extract the `PlainButton` to form a new component like `NavigationButton`.